### DISCUSSION TITLE
OTT Channels icon

### DISCUSSION THREAD

**Discussion by dreamealist** [twilio-labs/paste#3085] [Request: Icon] [answered]
### Description

<img width="743" alt="Screenshot 2023-03-08 at 2 17 12 PM" src="https://user-images.githubusercontent.com/13625142/224417710-33bf5706-4bc3-4d1b-af01-3686ea2b3e66.png">
These are OTT messaging channel icons that I'm currently using for Conversations API console features. From what I know, we don't currently have these included in the library.

I know many teams need cohesive channel icons for any channels related UX. Can we add these to the icon library? These are icons that I custom-created or put together from ones that have been used across other Twilio UIs

- SMS
- WhatsApp
- Chat
- Google Business Messages
- Facebook Business Messaging

Thank you!
Jiyeon Kang

### Link to icon

https://www.figma.com/file/KZjPSFNe4aLcnaxnR8L5eP/Conversations-Channels-Registration-and-Configuration?node-id=713%3A689082&t=rQyudCTjH50mPPqY-1

### Icon type

- [ ] UI icon
- [x] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @dreamealist ! Here are options/notes for each of the icons:

- SMS --> We recommend using SMSCapableIcon
- Chat --> We recommend using ProductChatIcon.
- WhatsApp, Google Business Messages, Facebook Business Messaging --> Unfortunately we don't add other companies' logos to Paste because they tend to have strict brand guidelines. Facebook Messenger, for example, only allows their logo in solid black or their gradient: https://about.meta.com/brand/resources/messenger/logo/ We recommend product teams manage these image assets on their own so they can adhere to the correct partner brand guidelines.

**Reply by dreamealist** [reply]
Thanks for your answer @serifluous. For SMS and Chat, I was exactly using those two that are already in the library. I will try to keep it consistent internally and encourage other designers to use them then -
